content/en/tracing/setup_overview/setup/php.md

```diff
@@ -439,6 +439,213 @@ To remove the PHP tracer:
 
 In the unusual event of an application crash caused by the PHP tracer, typically because of a segmentation fault, the best thing to do is obtain a core dump or a Valgrind trace and contact Datadog support.
 
+### Install debug symbols
+
+For the core dumps to be readable, debug symbols for the PHP binaries have to be installed in the system that runs PHP.
+
+To check if debug symbols are installed for PHP or PHP-FPM, the easiest approach is to use `gdb`.
+
+Install `gdb`:
+
+```
+apt|yum install -y gdb
+```
+
+Run `gdb` with the binary of interest. For example for PHP-FPM:
+
+```
+gdb php-fpm
+```
+
+If the `gdb` output contains a line similar to the text below, then debug symbols are already installed.
+
+```
+...
+Reading symbols from php-fpm...Reading symbols from /usr/lib/debug/path/to/some/file.debug...done.
+...
+```
+
+If the `gdb` output contains a line similar to the text below, then debug symbols have to be installed:
+
+```
+...
+Reading symbols from php-fpm...(no debugging symbols found)...done.
+...
+```
+
+Instructions follow in the next sections.
+
+#### Centos
+
+Install package `yum-utils` that provides the program `debuginfo-install`:
+
+```
+yum install -y yum-utils
+```
+
+Find the right package name for your PHP binaries, it can vary depending on the PHP installation method:
+
+```
+yum list installed | grep php
+```
+
+Install debug symbols. For example for package `php-fpm`:
+
+```
+debuginfo-install -y php-fpm
+```
+
+**Note**: if the repository that provides the PHP binaries is not enabled by default, it can be enabled when running the `debuginfo-install` command. For example:
```

Comment:
```suggestion
**Note**: If the repository that provides the PHP binaries is not enabled by default, it can be enabled when running the `debuginfo-install` command. For example:
```